src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$card-margin-right: 48px;
+$border-color: #f3f3f3;
+$blue-accent: #00acdc;
+$dark-text: #060b35;
+$dark-greyText: #616070;
+$white-color: #fff;
```

Comment:
Consider renaming the variable $white-color to $white for simplicity and consistency with other color variable names.